example/js/scalableTable.js

```diff
@@ -0,0 +1,840 @@
+angular.module('scalable-table', [])
```

Comment:
@trean The problem is that if you change something in the main script you have to change it in the example. It's not convenient. I think yes, you can use the file from the main directive.